Bottom: 1e34d849d02c5b7eb77f56c62577f589b680ce3e
Top:    4b9bcd7952b44a33be5d0d61d1d09a96ae9c3190
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-21 11:51:13 -0500

Update roles and privileges

Developers access to upload artifacts
Anonymous access to read artifacts


---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index 4445958a7..6e49eef7c 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -44,6 +44,8 @@ ldap_connections:
     ldap_user_subtree: false
     ldap_group_subtree: true
 
+nexus_anonymous_access: true
+
 nexus_privileges:
   - name: all-repos-read  # used as key to update a privilege
     description: 'Read & Browse access to all repos'
@@ -57,8 +59,9 @@ nexus_roles:
     name: developers
     description: All developers
     privileges:
-      - nx-search-read
       - all-repos-read
+      - nx-repository-view-*-*-*
+      - nx-search-read
     roles: []
 
 nexus_local_users:
